openupgradelib/openupgrade.py

```diff
@@ -499,6 +500,44 @@ def copy_fields_multilang(cr, destination_model, destination_table,
         )
 
 
+def remove_tables_fks(cr, *tables):
```

Comment:
I don't get why putting the argument with `*` and not simply pass an iterable

Reply:
In case you want to just drop a table, you don't need to create a list.

Well, nevermind, I'll change it as I see that passing lists around is the common usage in this lib.